### PR TITLE
 Update azure_rm_dnszone related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

 Update azure_rm_dnszone related document。

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_dnszone/azure_rm_dnszone_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_dnszone.py [hunk 1]
```diff
@@ -43,34 +43,33 @@
         required: true
     name:
         description:
-            - name of the DNS Zone.
+            - name of the DNS zone.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_dnszone.py]:
```suggestion
            - Name of the DNS zone.
```

File: lib/ansible/modules/cloud/azure/azure_rm_dnszone.py [hunk 2]
```diff
@@ -43,34 +43,33 @@
...
-            - This is a only when I(type) is C(private).
+            - This is a only when I(type=private).
             - Each element can be the name or resource id, or a dict contains C(name), C(resource_group) information of the virtual network.
         version_added: 2.8
         type: list
     resolution_virtual_networks:
         description:
             - A list of references to virtual networks that resolve records in this DNS zone.
-            - This is a only when I(type) is C(private).
+            - This is a only when I(type=(private).
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_dnszone.py]:
```suggestion
            - This is a only when I(type=private).
```

**Review by acozine — COMMENTED** [review]
Looks good, apart from the two typos I spotted.

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by acozine** [comment]
Thanks @Fred-sun!

**Pull request merged**